<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sudoku Game</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="grid_layout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>476</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="menu_new"/>
    <addaction name="menu_openfile"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_new">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="menu_openfile">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
